File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_api_result_page_plus_organization_member_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiResultPagePlusOrganizationMemberVO:
    """API调用结果"""
    data: PagePlusOrganizationMemberVO
    code: str
    msg: str
    request_id: str
    ip: str = None
    hostname: str = None
    error_name: str

File: sdkwork-backend-sdk-typescript/src/types/plus-api-result-page-plus-organization-member-vo.ts
import type { PagePlusOrganizationMemberVO } from './page-plus-organization-member-vo';

/** API调用结果 */
export interface PlusApiResultPagePlusOrganizationMemberVO {
  /** Response data */
  data: PagePlusOrganizationMemberVO;
  /** Response code: 2000=success, 4xxx=business failure, 5xxx=server error */
  code: string;
  /** Business message */
  msg: string;
  /** Request identifier */
  requestId: string;
  /** Client IP address */
  ip?: string;
  /** Server hostname */
  hostname?: string;
  /** Business error name */
  errorName: string;
}
